File: dixit-companion/src/main/java/fr/erban/dxitcompanion/game/player/PlayerBean.java
package fr.erban.dxitcompanion.game.player;

import java.io.Serializable;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Builder
@AllArgsConstructor
@Setter
public class PlayerBean implements Serializable {

    private String name;

    private int currentScore;

    private int scoreLastTurn;

    private List<TurnScore> scoresheet;

    private int nbGames;

    private int nbWins;

    private boolean persisted;
}

File: dixit-companion/src/main/java/fr/erban/dxitcompanion/game/player/PlayerBean.kt
package fr.erban.dxitcompanion.game.player

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class PlayerBean(
    val name: String,
    val currentScore: Int = 0,
    val scoreLastTurn: Int? = null,
    val scoresheet: List<TurnScore> = emptyList(),
    val nbGames: Int = 0,
    val nbWins: Int = 0,
    val persisted: Boolean = false
) : Parcelable
